libs/component-sample/src/lib/feature/side-navigation-sample/side-navigation-sample.component.html

```diff
@@ -2,17 +2,20 @@
     <sds-page-options class="sam-page__options">
         <sds-toolbar [expanded]="sideNavExpanded" #toolbar="sdsToolbar">
             <sds-accordion multi="true">
-                <sds-accordion-item [expanded]="sideNavExpanded">
+                <sds-accordion-item #navigationAccordion [expanded]="navigationExpanded">
```

Comment:
Styling is off in Chrome -- possibly incorrect class names
<img width="362" alt="Screen Shot 2020-06-16 at 11 10 07 AM" src="https://user-images.githubusercontent.com/63304927/84811675-1049a700-afc2-11ea-9efa-86ad8f722c6b.png">